frontend/src/operator/webui/src/app/view-pane/view-pane.component.ts

```diff
@@ -110,6 +141,7 @@ export class ViewPaneComponent implements OnInit, OnDestroy, AfterViewInit {
     this.displaysService.getDeviceVisibilities().pipe(
       map(deviceVisibilityInfos =>
         deviceVisibilityInfos.map(info => {
+          console.log('new info:', info);
```

Comment:
Can you please remove this message?

Reply:
Done.

I've also reverted the change in behavior re: custom icon panel assumed presence, and updated the comment. The justification I was using to change the behavior didn't make any sense, and it seems like it would be easy to make it work properly in a separate patch.